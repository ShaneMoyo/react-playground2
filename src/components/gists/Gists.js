import React from 'react'; 
import Search from './Search'; 
import GistList from './GistList'; 

export default function Gists() {

    return (
        <section>
            <h1>Gists</h1>
            <hr/>
            <Search/>
            <GistList/> 
        </section> 
    )
}