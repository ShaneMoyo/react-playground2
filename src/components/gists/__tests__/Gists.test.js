import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Gists from '../Gists'; 



describe('components', () => {
   
  it('should render AddImage component', () => {
    const wrapper = shallow(<Gists/>);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });

});