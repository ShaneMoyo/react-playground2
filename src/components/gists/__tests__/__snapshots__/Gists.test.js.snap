// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components should render AddImage component 1`] = `
<section>
  <h1>
    Gists
  </h1>
  <hr />
  <Search
    setGists={[Function]}
  />
  <GistList
    gists={Array []}
  />
</section>
`;
